drivers/video/fb.c

```diff
@@ -674,8 +674,15 @@ static int fb_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
         break;
 
       default:
-        gerr("ERROR: Unsupported IOCTL command: %d\n", cmd);
-        ret = -ENOTTY;
+        if (fb->vtable->ioctl)
```

Comment:
```suggestion
        if (fb->vtable->ioctl != NULL)
```
nit: Make the pointer validation more explicit by comparing to `NULL`, following the pattern from above cases.

Reply:
Done